source/agora/consensus/state/UTXOSet.d

```diff
@@ -70,7 +50,7 @@ public class UTXOSet
 
         Params:
             tx = the transaction
-            height = Height of the block where UTXO will be unlocked
+            height = current height of the block
```

Comment:
No, the documentation was correct. I tried to avoid the "current Height" as much as possible in docs, because it promotes the idea that we know the state of the network. You **never** know the current `Height`. You have a vision of the network, and perhaps you know what `Height` other nodes should be at, but you don't actually know (even if you ask them, they could lie).
Also, imagine you are doing catch-up from genesis, and there are 1000 blocks to catch-up. The first time you call this function, with a `Height == 1`, is it correct to say "current Height" ?

Reply:
Alright you can name it "local height". But the docs still lie. The utxo will be unlocked either at the next height or after unfreeze. Docs need fixing.